libretro-common/streams/file_stream.c

```diff
@@ -31,9 +31,10 @@
 #include "config.h"
 #endif
 
-#include <streams/file_stream.h>
 #define VFS_FRONTEND
```

Comment:
This #define must occur before including `file_stream.h` as it also includes `vfs_implementation.h` (prior to this PR). This becomes an issue when trying to use members of `stream->hfile` as not having `VFS_FRONTEND` defined results in `struct retro_vfs_file_handle` not being defined. This was breaking the wii and wiiu builds.